package/MDAnalysis/coordinates/base.py

```diff
@@ -781,20 +795,21 @@ def OtherWriter(self, filename, **kwargs):
             pass
         return core.writer(filename, **kwargs)
 
-    def _read_next_timestep(self, ts=None):  # pragma: no cover
+    @abc.abstractmethod
+    def _read_next_timestep(self, ts=None):
         # Example from DCDReader:
         #     if ts is None:
         #         ts = self.ts
         #     ts.frame = self._read_next_frame(etc)
         #     return ts
-        raise NotImplementedError(
-            "BUG: Override _read_next_timestep() in the trajectory reader!")
+        raise NotImplementedError()
```

Comment:
Leave the original error message?

Reply:
I changed the method to be an `abc.abstractmethod` so it's impossible to reach this code now